Spreadsheet/style.css

```diff
@@ -41,20 +41,19 @@ header {
 .main-top {
     display: flex;
     align-items: center;
-    border: 1px solid #01A9B4;
+    border-color: #01A9B4;
+    border-style: solid;
     border-width: 1px 0;
     width: 100%;
-
 }
 
 #fx img {
-    padding-top: 5px;    
+    padding-top: 5px;
 }
 
 #fx {
     padding: 0 10px;
-    /* border-right: 1px solid #e8eaed; */
-} 
+}
 
 #table-input {
```

Comment:
I didn't change this one but ideally namings should be consistent across the codebase: if you use camelCase in some classes/ids (`parentBoxColor`) then you need to use it for all of them (`tableInput`). Or inversely if you use kebab-case for some (`table-input`, `main-top`) then you need to use it for all (`parent-box-color`).

It seems like a small detail but it actually really helps with the overall readability of the code.

I personally like to use camelCase for css selectors because of the fact that it's also being used in the javascript part of the code for variables and visually close to PascalCase used for object classes in javascript/React. This way the reader will mostly see the same naming style across almost the whole codebase. Otherwise they have to mentally switch between camel/PascalCase and kebab-case all the time and it can become tiring.

Just a personal preference though, it's a very opinionated topic and a lot of people will like to use either kebab-case and snake_case for css selectors, but that's also due to historical reasons and what language they used to code in before moving into web. I recently started seeing camelCase used more and more for selectors so I guess I'm not the only one who sees the value in using it.